src/components/views/auth/PasswordLogin.js

```diff
@@ -193,7 +193,9 @@ class PasswordLogin extends React.Component {
                     name="username" // make it a little easier for browser's remember-password
                     key="username_input"
                     type="text"
-                    label={SdkConfig.get().disable_custom_urls ?
+                    label={SdkConfig.get().username_description ?
+                        SdkConfig.get().username_description :
+                        SdkConfig.get().disable_custom_urls ?
```

Comment:
Doubly-nested ternarys should never be a thing. Please break this out into full if-statements.